frameworks/digital-outcomes-and-specialists-2/questions/brief-responses/niceToHaveRequirements.yml

```diff
@@ -1,4 +1,5 @@
 name: Nice-to-have requirements
+short_name: Nice-to-have skills and experience
```

Comment:
Personally I wouldn't bother making changes to the DOS2 questions as we will copy them all again from DOS 1 at a later point for which the copy is still being changed.